games/devtest/mods/testabms/changable.lua

```diff
@@ -0,0 +1,48 @@
+local abm_first_run = false;
+
+minetest.register_chatcommand("changableABM", {
+	params = "enable | disable | interval_1 | interval_5",
+	description = "Active/Deactivate changable ABM.",
+	func = function(name, param)
+		if param == "enable" then
+			minetest.change_abm("testchangableabm:abm", {
+					interval = 30.0,
+				})
+			abm_first_run = true;
+			return true, "ABM has been enabled."
+		elseif param == "interval_1" then
+			minetest.change_abm("testchangableabm:abm", {
+					interval = 1.0,
+				})
+			abm_first_run = true;
+				return true, "ABM has been enabled with interval 1."
+		elseif param == "interval_5" then
+			minetest.change_abm("testchangableabm:abm", {
+					interval = 5.0,
+				})
+			abm_first_run = true;
+				return true, "ABM has been enabled with interval 5."
+		elseif param == "disable" then
+			minetest.change_abm("testchangableabm:abm", {
+					interval = -30.0,
+				})
+			return true, "ABM has been disabled."
+		else
+			return false, "Check /help changableABB for allowed parameters."
+		end
+	end,
+})
```

Comment:
rather than this I'd add an unit test

Reply:
We have no unit tests for ABM so far which can be extended.
Would it be better to do some ABM unit tests in a separate PR?

Reply:
it's not easy to thoroughly test but I was thinking just having an ABM registered and then calling `change_abm` a few times.
In the devtest Lua unittests, to be clear.

Reply:
I added unittest... but it effectively checks only changes in `core.registered_abms`